xbmc/filesystem/AddonsDirectory.cpp

```diff
@@ -62,12 +62,27 @@ const std::array<TYPE, 5> infoProviderTypes = {
   ADDON_SCRAPER_TVSHOWS,
 };
 
+const std::vector<TYPE> lookAndFeelTypes = {
```

Comment:
I *highly* doubt that since there are 6 items. What is slow here is the constant creating and destructing of addons. But sure, can do for readability;)